security-lexicon/cloud-native-security-lexicon.md

```diff
@@ -0,0 +1,1330 @@
+                <li>key management</li>
+                <li>use of cryptographic and sensitive data material, for example, performing encryption or digital signature functions</li>
+                <li>offloading application servers for complete asymmetric and symmetric cryptography.</li>
+            </ol>
+        </td>
+</tr>
+<tr>
+        <th >References</th>
+        <td> https://csrc.nist.gov/glossary/term/hardware_security_module_hsm  </td>
+</table>
+<br />
+<br />
+<table style="width:100%">
+    <tr>
+        <th colspan="2" id="hsm-tpm">HSM vs TPM</th>
+    </tr>
+    <tr>
+        <th >Definition</th>
+        <td> A Trusted Platform Module (TPM) is a hardware chip on the computer’s motherboard that stores cryptographic
+            keys used for encryption. Many laptop computers include a TPM, but if the system doesn’t include it, it is
+            not feasible to add one. Once enabled, the Trusted Platform Module provides full disk encryption
+            capabilities. It keeps hard drives locked, or sealed, until the system completes a system verification or
+            authentication process.
+            <br /><br />
+            The TPM includes a unique RSA key burned into it, which is used for asymmetric encryption. Additionally, it
+            can generate, store, and protect other keys used in the encryption and decryption process.
+            <br /><br />
+            A hardware security module (HSM) is a security device you can add to a system to manage, generate, and
+            securely store cryptographic keys.
+            <br /><br />
+            High performance HSMs are external devices connected to a network using <!-- cspell:disable -->TCP/IP. Smaller HSMs come as
+            expansion cards you install within a server, or as devices you plug into computer ports. Recommendation is
+            to use HSM’s which are <!-- cspell:disable -->FIPS 140-2 compliant.
+            <br /><br />
+            One of the noteworthy differences between the two is that HSMs are removable or external devices. In
+            comparison, a TPM is a chip embedded into the motherboard. You can easily add an HSM to a system or a
+            network, but if a system didn’t ship with a <!-- cspell:disable -->TPM, it’s not feasible to add one later. Both provide secure
+            encryption capabilities by storing and using RSA keys.
+        </td>
+    </tr>
+    <tr>
+        <th >Terms</th>
+        <td> null </td>
+    </tr>
+    <tr>
+        <th >Organizational Use</th>
+        <td> null </td>
+    </tr>
+    <tr>
+        <th >References</th>
+        <td> https://blogs.getcertifiedgetahead.com/tpm-hsm-hardware-encryption-devices/ </td>
+</table>
+<br/>
+<br/>
+<table style="width:100%">
+<tr>
+	<th colspan="2" id="certificate-authority">Certificate authority </th>
```

Comment:
Maybe we want to move this definition closer to:
https://github.com/cncf/tag-security/blob/0027440f3cbdf6fbd73c75ab95a642d7bc59fe2d/security-lexicon/cloud-native-security-lexicon.md#certificates

Reply:
Done